src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -1033,9 +1034,14 @@ cdef class DrawFuncs:
                                      object,  # user_data
                                  ], None],
                                  user_data: object = None) -> None:
-        self._move_to_func = func
+        cdef hb_draw_move_to_func_t func_p
+        if PyCapsule_IsValid(func, NULL):
+            func_p = <hb_draw_move_to_func_t>PyCapsule_GetPointer(func, NULL)
```

Comment:
For completeness sake, this should probably set `self._move_to_func` to None, just in case this call is overriding a previous call.